packages/web/src/react-query/token/request.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosClient } from "@common/clients/network-client/axios-client";
+import { TokenListResponse, TokenPriceListResponse } from "@repositories/token";
+
+const API_URL = process.env.NEXT_PUBLIC_API_URL;
+
+const request = new AxiosClient(API_URL);
+
+export const getTokens = async (): Promise<TokenListResponse> => {
```

Comment:
Fetch logic is already implemented in the token repository. 
Why did you create a new function?